### ISSUE TITLE
New Title and text for Tutorials tile on main page

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oeplatform#1174]
As we reworked the tutorial section, I suggest changing the title and text of the tile attached below.

Suggested Title: _Trainings & Tutorials_
Suggested text: _Our training and tutorial section provides whole training courses as well as dedicated tutorials covering important topics around the Open Energy Family (OEF) tools and the Open Energy Platform (OEP)._ 

![grafik](https://user-images.githubusercontent.com/36844029/224969823-b4500e5d-744e-465a-8284-85bd3746f285.png)

**Comment by u-mueller** [comment]
I like your suggestion and approve the more factual language for describing our tools.

**Comment by han-f** [comment]
New suggestion, due to lead jour fixe insights of 2023-03-15:
Suggested Title: Open Energy Academy (OEA)
Suggested text: The OEA provides whole courses as well as dedicated tutorials covering important topics around the Open Energy Family (OEF) tools and the Open Energy Platform (OEP). You will also find short answers to short questions.

**Comment by u-mueller** [comment]
Can we say "short answers to urgent questions" or something like this?

**Comment by han-f** [comment]
@u-mueller - yes we can, good idea. Updated suggestion:
Suggested Title: _Open Energy Academy (OEA)_
Suggested text: _The OEA provides courses as well as dedicated tutorials covering important topics around the Open Energy Family (OEF) tools and the Open Energy Platform (OEP). You will also find short answers to urgent questions._

**Comment by wingechr** [comment]
I think the title should just be `Academy`, so that it looks better together with the other links:

![grafik](https://user-images.githubusercontent.com/4265368/226109912-dbfdedc3-21fd-43c7-9d9a-36aa9f368bfb.png)

**Comment by u-mueller** [comment]
> I think the title should just be `Academy`, so that it looks better together with the other links:
> 
> ![grafik](https://user-images.githubusercontent.com/4265368/226109912-dbfdedc3-21fd-43c7-9d9a-36aa9f368bfb.png)

I agree, "Academy" as title on the start page would be more consistent.

**Comment by wingechr** [comment]
implemented in latest release
